configure.ac

```diff
@@ -280,14 +274,14 @@ LIBS="$LIBS $ELL_LIBS"
 dnl l_genl_msg_get_extended_error() was introduced in ELL v0.31.
 AC_CHECK_FUNC([l_genl_msg_get_extended_error],
               [AC_DEFINE([HAVE_L_GENL_MSG_GET_EXTENDED_ERROR],
-	                 [],
-			 [ELL has l_genl_msg_get_extended_error()])])
+                         [],
```

Comment:
Tab to space conversion? Wasn't sure if it was intentional, but looks like everything has now been normalized to spaces, so that seems fine.

Reply:
The spaces were intended.  It seems there left over tabs in this file from previous commits that I missed.